### ISSUE TITLE
Blank page when loading from /Admin

### ISSUE THREAD

**Issue by cxbxmxcx** [IdentityServer/IdentityServer3.Admin#35]
Noticed that this was already reported with an Angular error 5 days ago, but in my case I see no error.

![image](https://cloud.githubusercontent.com/assets/10665060/14472489/9f001e7c-00b0-11e6-8900-633c152edb7d.png)

Pulled the latest code today (4/12/2016) and restored all Nuget packages, including updating IdentityServer3.

Any thoughts/recommendations?

Cheers,

M

**Comment by iangriffin** [comment]
I pulled today too, no issues here. Just opened it, built it and ran it. Check the network tab. Are you seeing the css, javascript and `Templates.navbar.html` files loading? Maybe that will give some indication of the issue.

**Comment by iBoonz** [comment]
Maybe chrome cashing? Try Ctrl F5, for hard refresh

**Comment by cxbxmxcx** [comment]
I managed to get it working, thanks for the quick replies.  Those responses did help.

I think my nuget packages was corrupted possibly on the way I initially opened the project?

Here is what I did:

- deleted the old project and downloaded again

- installed IdentityServer3 Nuget package on Host project

- Had some issues with the Test project, so I just removed it from the solution

- Restored the Nuget packages on Core

After that it works fine.

Thanks,

M
